### ISSUE TITLE
SendTextPlus sends line and then goes to console

### ISSUE THREAD

**Issue by miguelroboso** [randy3k/SendTextPlus#27]
I thought it was supposed to stay in the source, instead every time I run a line, the focus does not shift back to the source window.

How do I change this behavior?

I am on Mac OS 10.11, Sublime Text 3, and the code comes from R-Box.

I tried sending code to Terminal, Safari-RStudio, and to REPL, and the behavior is the same.

**Comment by randy3k** [comment]
It is strange. The focus should remain at sublime on Mac OS X....

**Comment by miguelroboso** [comment]
Yes, that is what it was doing previously. Any idea of what should cause it and/or how can I solve it?

**Comment by randy3k** [comment]
Did you figure it out? I really have no clue.

**Comment by miguelroboso** [comment]
How do you think I can figure it out? Is there any log I can take a look at?
By the way, this happens also on my Ubuntu machine.

In the old sendtext there was a file allowing me to change the key bindings, is there a similar file in SendTextPlus?

**Comment by hafen** [comment]
I just updated ST3 and have the problem that I *want* focus to stay on terminal after sending text.  I can't seem to find the setting for this.  Can this be done?

**Comment by randy3k** [comment]
@hafen , what focus should remain at Sublime Text. Currently, there is no way to switch focus.

**Comment by randy3k** [comment]
@miguelroboso 

I don't think it is an issue of SendTestPlus itself, what else packages have you installed?

The key binds are defined in the keybind files, e.g. https://github.com/randy3k/SendTextPlus/blob/master/Default%20(OSX).sublime-keymap

**Comment by miguelroboso** [comment]
@randy3k 

I have installed only R-box and SublimeREPL (with all the dependencies that come with them).

In the "Packages" dir in ST I have:

-SublimeREPL
-bz2

-User->R-Box

And nothing more

As I wrote, though, it happens with my ubuntu machine too, and that was a super-fresh installation of R-box and sublimerepl.

**Comment by hafen** [comment]
Thanks.  Switching focus would be a nice option to have.  I just forked and modified to suit my needs.  If I had time, I would propose a clean / general / configurable solution and send a PR, but this suits me for now.
